_fossfund/2-rust-analyzer.md

```diff
@@ -4,8 +4,8 @@ logo: rust-analyzer.svg
 projectUrl: https://rust-analyzer.github.io/
 linkText: "rust-analyzer.github.io"
 description: "rust-analyzer is an implementation of Language Server Protocol for the Rust programming language. It provides features like completion and goto definition for many code editors, including VS Code, Emacs and Vim."
-processing: "This sponsorship is currently being processed.  The selection process completed in July 2020."
+#processing: "This sponsorship is currently being processed.  The selection process completed in July 2020."
 context: "There is a large Rust fanbase at Microsoft, and engineers using Rust find themselves using rust-analyzer every day. It's often paired with Visual Studio Code by productive Rust engineers."
-awarded: true # in processing
+awarded: true #in processing
```

Comment:
You can just remove those lines altogether (instead of commenting them out) thanks for catching that!